ext/fetch/proxy.rs

```diff
@@ -0,0 +1,602 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+use std::env;
```

Comment:
Could you please add a module doc here regarding that this is temporary until a proxy utility is provided by `hyper` crates?